packages/playwright-test/types/test.d.ts

```diff
@@ -2620,7 +2619,7 @@ export type VideoMode = 'off' | 'on' | 'retain-on-failure' | 'on-first-retry';
  * ```
  *
  */
-export interface PlaywrightTestOptions {
+export interface PlaywrightTestOptions extends PlaywrightTest.TestOptions {
```

Comment:
This is an odd direction of extension. If user overrides `acceptDownload: string` in their global, what will TS compiler say? I think it will yell about `PlaywrightTestOptions`, but we want it to say `PlaywrightTest.TestOptions` is wrong.

Reply:
Error would still be actionable i think